### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by KochTobi**
This PR adds tests for the REST controller

### CODE REVIEW

File: src/test/java/life/qbic/subscriptions/SubscriptionControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+    mockMvc
+        .perform(
+            get("/subscription/cancel")
+                .contentType(MediaType.APPLICATION_JSON)
+                .characterEncoding(StandardCharsets.UTF_8)
+                .content(json))
+        .andExpect(status().is(401));
+  }
+}
```
Comment (sven1103) [inline src/test/java/life/qbic/subscriptions/SubscriptionControllerTest.java]:
```suggestion
}

```

Comment (KochTobi) [reply src/test/java/life/qbic/subscriptions/SubscriptionControllerTest.java]: Why is this important?

Comment (sven1103) [reply src/test/java/life/qbic/subscriptions/SubscriptionControllerTest.java]: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Comment (sven1103) [reply src/test/java/life/qbic/subscriptions/SubscriptionControllerTest.java]: following the POSIX standard, L144 is not a line. That is also why Github thankfully highlights this for us.

Comment (sven1103) [reply src/test/java/life/qbic/subscriptions/SubscriptionControllerTest.java]: shared this also with the team now

**Review by sven1103 — APPROVED** [review]
Great tests!

**Pull request merged**